Software/Source/Bursttest.py

```diff
@@ -0,0 +1,283 @@
+import csv
+import json
+
+Dbg_Pkt=0
+Acc_Pkt=0
+Temp_Pkt=0
+Batt_Pkt=0
+Brst_Pkt=0
+k=0
+l=0
+m=0
+n=0
+o=0
+macid=[]
+count=[]
+count1=[]
+slno=0
+#Function to check if macid is already present in array.
+def checkmacid(file_mac):
+
+	global macid
+	n = -1
+	for i in range (len(macid)):
+		if macid[i] == file_mac:
+			n = i
+	return n
+
+#Function to return the arry of macid
+def comparemacid():	
+	f=open("Non-Connected_Data_G2.txt","r")
+	while True:	
+		content=f.readline()	
+		if content:
+			file_obj = json.loads(content)
+			file_mac = file_obj["SensorUUID"]
+		else: 
+			break	#if no content
+		n=checkmacid(file_mac)
+		if n ==-1:
+			macid.append(file_mac)
+	print(macid)
+	return(macid)
+
+# Function to create and write to the macid text file.
+def writefile(file_mac):
+
+	myfile=open("Non-Connected_Data_G2.txt","r")
+	while True:
+		content=myfile.readline()
+		# print(content)
+		if content:
+			file_obj = json.loads(content)
+			mac_id=file_obj["SensorUUID"]
+			if mac_id == file_mac:
+				file = open(str(mac_id)+".txt","a")
+				py_obj = json.dumps(file_obj)
+				file.write(py_obj+"\n")
+				file.close()
+		else:
+			break

```

Comment:
if we have blank space in between lines of data in the file, then this logic will exit in between leaving the remaining data unread. Work on the logic to overcome this